### ISSUE TITLE
Allow embedded per-file rules.

### ISSUE THREAD

**Issue by shannonmoeller** [CSSLint/csslint#336]
I'd like to be able to specify rules on a per-file basis by embedding the options directly in the CSS file as a comment, similar to jslint and jshint:

``` css
/*csslint --ignore=adjoining-classes,box-sizing --error=universal-selector */
```

Or, to follow jslint convention (defaults to warn, false is ignore, true is error):

``` css
/*csslint adjoining-classes:false, box-sizing:false, universal-selector:true */
```
